recipe/meta.yaml

```diff
@@ -82,6 +82,7 @@ outputs:
         - pytest-subtests >=0.11.0
         - pytest-vcr >=1.0.2
         - pytest-xdist >=3.5.0
+        - pytorch != 2.3.2                     # [aarch64]
```

Comment:
Should this be equal to this version?  (Or `!= 2.4.0` if we are optimistic that future versions fix the problem).

Reply:
Normally I would be optimistic. Should we pin to exactly one version instead?

Reply:
I'm not sure, but I can say that `pytorch !=2.3.2` is definitely not the correct constraint.